views/Nav.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import { View, Text, Modal, Button, Picker } from "react-native";
+
+function Nav(props) {
+
+  const playerDifficultyRender = difficulty => {
+    if (difficulty === 3) {
+      return "EASY"
+    } else if (difficulty === 2) {
+      return "MEDIUM"
+    } else {
+      return "HARD"
+    }
+  }
+
+  return (
+    <Modal visible={props.hidden}>

```

Comment:
Can you center this on the screen? I can't hit all the buttons on my iPhone because of the shape of the iPhone screen

Reply:
Right, makes sense.